htdocs/core/class/commonobject.class.php

```diff
@@ -10733,7 +10734,6 @@ public function setSignedStatusCommon($user, $status, $notrigger = 0, $triggerco
 			}
 
 			if (!$error) {
-				$this->status = $status;
```

Comment:
Should not be removed but replace by 
$this->signed_status = 1;

Reply:
I removed this and added it to the `setSignedStatus` method of `fichinter.class.php` because the `$signed_status` property is defined in that class and not in `commonobject.class.php`. Setting it here means accessing it through magic method. If that isn't a problem I can add it back here. Maybe moving `signed_status` to `CommonObject` class could be discussed in another PR ?